### PR TITLE
Jenkins triage fixes

### PR DESCRIPTION
**Pull request by dheyay**
## Why is this needed?

Fixes for integration test issues I found during the jenkins triage.

## Checklist

 - [x] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] Changes here need to be documented, and this was done in: 

## Does this PR require extra reviews?

 - [ ] Yes
 - [ ] No

### CODE REVIEW

File: uaclient/cli/enable.py [hunk 1]
```diff
@@ -86,6 +94,32 @@ def prompt_for_dependency_handling(
...
 
+    if (
+        service in enabled_service_names
+        and variant
+        and variant not in enabled_service_variants
+    ):
+        variant_title = entitlements.get_title(cfg, service, variant)
+        enabled_variant_title = entitlements.get_title(
+            cfg, service, enabled_service_variants[0]
+        )
```
Comment (orndorffgrant) [inline uaclient/cli/enable.py:105]: variant names without the associated service name is missing information. There could be a "generic" variant for more than one service.

I suggest something like the following:
```suggestion
    variant_enabled = next(
        (s for s in enabled_services if s.name == service and s.variant_enabled and s.variant_name != variant), None
    )
    if variant_enabled is not None:
        to_be_enabled_title = entitlements.get_title(cfg, service, variant)
        enabled_variant_title = entitlements.get_title(cfg, service, variant_enabled.variant_name)
```

Comment (dheyay) [reply uaclient/cli/enable.py:105]: Understood, the use of `next` is also better than the indexing

**Comment by dheyay** [comment]
Updated! @athos-ribeiro

**Review by orndorffgrant — APPROVED** [review]
Thanks @dheyay This looks good. The prompts worked correctly after this change

**Review by athos-ribeiro — APPROVED** [review]

**Pull request merged**